CHANGELOG-1.0.md

```diff
@@ -0,0 +1,120 @@
+# Release notes for 1.0.0
+
+[Documentation](https://kubernetes-csi.github.io)
+
+# Changelog since 0.5.0
+
+## Changes by Kind
+
+### Feature
+
+- Adding configurable QPS and Burst to k8s client and use a separate client for leader election. ([#91](https://github.com/kubernetes-csi/external-resizer/pull/91), [@RaunakShah](https://github.com/RaunakShah))
+- Do not call controller-expand if volume is in-use and can not be expanded while in-use ([#86](https://github.com/kubernetes-csi/external-resizer/pull/86), [@gnufied](https://github.com/gnufied)) . This feature requires watching for all pods in the cluster and hence can be expensive operation. It can be turned off by using `--handle-volume-inuse-error` flag([#89](https://github.com/kubernetes-csi/external-resizer/pull/89), [@saikat-royc](https://github.com/saikat-royc)).
```

Comment:
should we add action required to this note so it shows up as an urgent upgrade note?